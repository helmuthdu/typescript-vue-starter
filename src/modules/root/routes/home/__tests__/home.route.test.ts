import { shallowMount, Wrapper } from '@vue/test-utils';
import HomeRoute from '../home.route.vue';

describe('Home/Route -> Home', () => {
  let wrapper: Wrapper<HomeRoute>;

  beforeEach(() => {
    wrapper = shallowMount(HomeRoute);
  });

  it('should renders with props', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
