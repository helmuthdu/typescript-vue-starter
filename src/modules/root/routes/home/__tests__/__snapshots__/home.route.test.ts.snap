// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home/Route -> Home should renders with props 1`] = `
<section>
  <hr> <a href="https://vuejs.org/" target="_blank">vue Documentation</a> <br> <a href="https://github.com/vue/vue.js" target="_blank">vue GitHub</a></section>
`;
